### PR TITLE
feat: Support PHP v2 and v4 schema payloads

### PR DESCRIPTION
**Pull request by keelerm84**

### CODE REVIEW

File: sdktests/php_events_all.go [hunk 1]
```diff
@@ -34,7 +34,14 @@ func doPHPEventRequestTests(t *ldtest.T) {
 		}))
 
 	eventTests.RequestMethodAndHeaders(t, sdkKey,
-		Header("X-LaunchDarkly-Event-Schema").Should(m.Equal(phpEventSchema)))
```
Comment (cwaldren-ld) [inline sdktests/php_events_all.go:37]: In this case, should `phpEventSchema` be renamed something like `php(Deprecated|Legacy|Whatever)EventSchema` to indicate that it's no longer the latest and greatest?

File: sdktests/php_events_all.go [hunk 2]
```diff
@@ -34,7 +34,14 @@ func doPHPEventRequestTests(t *ldtest.T) {
...
+		m.AnyOf(
+			Header("X-LaunchDarkly-Event-Schema").Should(m.Equal(phpEventSchema)),
+			m.AllOf(
+				Header("X-LaunchDarkly-Event-Schema").Should(m.Equal(currentEventSchema)),
+				Header("X-LaunchDarkly-Summarized").Should(m.Not(m.BeNil())),
```
Comment (cwaldren-ld) [inline sdktests/php_events_all.go:41]: Should be `X-LaunchDarkly-Unsummarized ?

File: sdktests/php_events_all.go (hunk 1 above)
Comment (keelerm84) [reply sdktests/php_events_all.go:37]: Done!

File: sdktests/php_events_all.go (hunk 2 above)
Comment (keelerm84) [reply sdktests/php_events_all.go]: Done

**Review by cwaldren-ld — APPROVED** [review]

**Pull request merged**